templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -23,7 +23,7 @@ description: A complete ToDo app with C# API and Azure Cosmos DB SQL API for sto
 # React Web App with C# API and MongoDB on Azure
 
 [![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=446558288&machine=standardLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2)
-[![Open in Remote - Containers](https://img.shields.io/static/v1?style=for-the-badge&label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/todo-csharp-cosmos-sql)
+[![Open in Dev Container](https://img.shields.io/static/v1?style=for-the-badge&label=Dev+Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/todo-csharp-cosmos-sql)
```

Comment:
Updated now to use that new codespaces api to create codespaces :)

Reply:
@jongio , using the `url` you provided (`https://codespaces.new/owner/repo`) basically redirects the user to the one we currently have (`https://github.com/codespaces/new`).  And since we are not passing the codespace settings (like before), user is prompt to choose settings like:

![image](https://github.com/Azure/azure-dev/assets/24213737/f36132d7-3fb2-4620-a9a7-733ca596b803)


In previous approach, we set all those settings with a query like `hide_repo_select=true&ref=main&repo=XXXXXX&machine=standardLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2`

Do you still want to switch to this other API?

Reply:
Yeah, that is the approach the codespaces team would like us to take.